crates/sui-core/src/epoch/tests/reconfiguration_tests.rs

```diff
@@ -163,16 +157,11 @@ async fn test_start_epoch_change() {
 }
 
 #[tokio::test]
+#[ignore]
```

Comment:
Do these changes break this test?

Reply:
thanks for catching this. i did test this locally but forgot to unignore it